dbt/adapters/athena/impl.py

```diff
@@ -522,21 +522,27 @@ def swap_table(self, src_relation: AthenaRelation, target_relation: AthenaRelati
         # if source table has partitions we need to delete and add partitions
         # it source table hasn't any partitions we need to delete target table partitions
         if target_table_partitions:
-            glue_client.batch_delete_partition(
-                DatabaseName=target_relation.schema,
-                TableName=target_relation.identifier,
-                PartitionsToDelete=[{"Values": i["Values"]} for i in target_table_partitions],
-            )
+            for partition_batch in get_chunks(target_table_partitions, 100):
```

Comment:
it makes sense to move constant `100` as constant in `config` module as it probably will be reused in future

Reply:
that was actually `25`, I accidentally changed to `100` apparently. 
So you still want that constant? Isn't it a bit overkill?

But if you really want, I can do it, just how to name it? `HUNDRED` vs. `BATCH_CREATE_PARTITION_LIMIT` ? depends on how you want to reuse it. 
maybe let's create it when we need to reuse it

Reply:
it's general Athena limitation. We may call it `ATHENA_PARTITIONS_LIMIT`

Reply:
Ah you mean the limit for CTAS, but in this case its API related. The limits differ, [BatchCreatePartition has a limit of 100](https://docs.aws.amazon.com/glue/latest/webapi/API_BatchCreatePartition.html), [BatchDeletePartition has a limit of 25](https://docs.aws.amazon.com/glue/latest/webapi/API_BatchDeletePartition.html). I don't think its really related to the CTAS-partition limit, just API payload restriction.

Reply:
sure, I see. It would be great to add such constant `BATCH_CREATE_PARTITION_LIMIT` to `config` module or as `AthenaAdapter` class constant and refer to official documentation to make the code clear and simple to understand